docs/node_software.md

```diff
@@ -28,10 +29,10 @@ At this stage, we recommend using Hornet since it has several optional features
 
 ## Node Software
 
-- [hornet](https://wiki.iota.org/hornet/welcome)
-- [bee (branch: chrysalis-pt-2)](https://github.com/iotaledger/bee/tree/chrysalis-pt-2)
+- [hornet](https://wiki.iota.org/hornet/welcome).
```

Comment:
```suggestion
## Hornet

- [Official GitHub Repository](https://github.com/gohornet/hornet)
- [Documentation](https://wiki.iota.org/hornet/welcome).
```

Reply:
If you want to go down this road, you could also refactor the previous text into each header.